Dice-Roll-Simulator/dice_roll_simulator.py

```diff
@@ -0,0 +1,49 @@
+import tkinter
+from PIL import Image, ImageTk
+import random
+
+# toplevel widget which represents the main window of an application
+root = tkinter.Tk()
+root.geometry('400x400')
+root.title('Roll the Dice')
+
+# Adding label into the frame
+l0 = tkinter.Label(root, text="")
+l0.pack()
+
+# adding label with different font and formatting
+l1 = tkinter.Label(root, text="Dice Rolling Simulator", fg = "white",
+        bg = "black",
+        font = "Helvetica 16 bold italic")
+l1.pack()
+
+# images
+dice = ['die1.png', 'die2.png', 'die3.png', 'die4.png', 'die5.png', 'die6.png']
```

Comment:
```suggestion
dice = ['./Dice-Roll-Simulator/imgs/die1.png', './Dice-Roll-Simulator/imgs/die2.png', './Dice-Roll-Simulator/imgs/die3.png', './Dice-Roll-Simulator/imgs/die4.png', './Dice-Roll-Simulator/imgs/die5.png', './Dice-Roll-Simulator/imgs/die6.png']
```

Reply:
Or could even use list comprehension
```suggestion
dice_names = ['die1.png', 'die2.png', 'die3.png', 'die4.png', 'die5.png', 'die6.png']
dice_path = [os.path.join(os.getcwd() , 'imgs' , ele) for ele in dice_names]
```